tests/commands/arch.py

```diff
@@ -0,0 +1,47 @@
+"""
+Arch commands test module
+"""
+
+from tests.base import RemoteGefUnitTestGeneric
+from tests.utils import ARCH
+import pytest
+
+
+class ArchCommand(RemoteGefUnitTestGeneric):
+    """Generic class for command testing, that defines all helpers"""
+
+    def setUp(self) -> None:
+        return super().setUp()
+
+    @pytest.mark.skipif(ARCH != "x86_64", reason=f"Skipped for {ARCH}")
+    def test_cmd_arch_get(self):
+        gdb = self._gdb
+
+        res = gdb.execute("arch get", to_string=True)
+        self.assertIn(" Architecture(X86, 64, LITTLE_ENDIAN)", res)
```

Comment:
we can do this later (I am sure many tests do this 'wrong' but with the pytest runner we get better errors logs if you do
`assert "..." in res`
and you can even explain
`assert "..." in res, "Arch should be x86 64"`